src/components/navBar/navBar.js

```diff
@@ -90,7 +90,7 @@ angular.module('material.components.navBar', ['material.core'])
  * Exactly one of the `md-nav-click`, `md-nav-href`, or `md-nav-sref` attributes are required
  * to be specified.
  *
- * @param {string=} aria-label Adds alternative text for accessibility.
+ * @param {string=} aria-label Adds alternative text for accessibility to the `<md-button>` element.
```

Comment:
I think that my main point was that we need to rename the API from `aria-label` to `nav-item-aria-label`.

For the text, let's use "Allows setting or overriding the label that is announced by a screen reader for the nav item's button. If this is not set, the nav item's transcluded content will be announced. Make sure to set this if the nav item's transcluded content does not include descriptive text, for example only an icon."